packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -313,33 +311,52 @@ function buildTroubleshooterData({
   return troubleshooterData;
 }
 
+export type PlatformParams = {
+  platformClientId?: string;
+  platformCancelUrl?: string;
+  platformBookingUrl?: string;
+  platformRescheduleUrl?: string;
+  platformBookingLocation?: string;
+};
+
+export type BookingHandlerInput = {
+  bookingData: Record<string, any>;
```

Comment:
This input is untyped just like req.body was. 

We generate a schema below in the `bookingDataSchemaGetter` callback or default to `getBookingDataSchema`